test-app/tests/integration/components/textarea-field-test.gts

```diff
@@ -0,0 +1,181 @@
+    await render(<template>
+      <TextareaField @label="Label" @error="Error text" data-textarea />
+    </template>);
+
+    let error = find('[data-error]');
+
+    assert.dom(error).hasText('Error text');
+    assert.dom(error).hasAttribute('id');
+
+    let errorId = error?.getAttribute('id') || '';
+    assert.ok(errorId, 'Expected error ID to be truthy');
+
+    let describedby =
+      find('[data-textarea]')?.getAttribute('aria-describedby') || '';
+    assert.ok(
+      describedby.includes(errorId),
+      'Expected errorId to be included in the aria-describedby'
+    );
+
+    assert.dom('[data-textarea]').hasAttribute('aria-invalid', 'true');
+  });
+
+  test('it sets aria-describedby when both a hint and error are provided using the hint and error IDs', async function (assert) {
+    await render(<template>
+      <TextareaField
+        @label="Label"
+        @error="Error text"
+        @hint="Hint text"
+        data-textarea
+      />
+    </template>);
+
+    let errorId = find('[data-error]')?.getAttribute('id') || '';
+    assert.ok(errorId, 'Expected error ID to be truthy');
+
+    let hintId = find('[data-hint]')?.getAttribute('id') || '';
+    assert.ok(hintId, 'Expected hint ID to be truthy');
+
+    assert
+      .dom('[data-textarea]')
+      .hasAttribute('aria-describedby', `${errorId} ${hintId}`);
+  });
+
+  test('it disables the textarea using `@isDisabled`', async function (assert) {
+    await render(<template>
+      <TextareaField @label="Label" @isDisabled={{true}} data-textarea />
+    </template>);
+
+    assert.dom('[data-textarea]').isDisabled();
+    assert.dom('[data-textarea]').hasClass('text-disabled');
+  });
+
+  test('it spreads attributes to the underlying textarea', async function (assert) {
+    await render(<template>
+      <TextareaField
+        @label="Label"
+        placeholder="Placeholder text"
+        data-textarea
+      />
+    </template>);
+
+    assert
+      .dom('[data-textarea]')
+      .hasAttribute('placeholder', 'Placeholder text');
+  });
+
+  test('it sets the value attribute via `@value`', async function (assert) {
+    await render(<template>
+      <TextareaField @label="Label" @value="tony" data-textarea />
+    </template>);
+
+    assert.dom('[data-textarea]').hasValue('tony');
+  });
+
+  test('it calls `@onChange` when input is received', async function (assert) {
+    assert.expect(6);
+
+    let handleChange = (value: string, e: Event | InputEvent) => {
+      assert.strictEqual(value, 'test', 'Expected input to match');
+      assert.ok(e, 'Expected `e` to be available as the second argument');
+      assert.ok(e.target, 'Expected direct access to target from `e`');
+      assert.step('handleChange');
+    };
+
+    await render(<template>
+      <TextareaField @label="Label" @onChange={{handleChange}} data-textarea />
+    </template>);
+
+    assert.verifySteps([]);
+
+    await fillIn('[data-textarea]', 'test');
```

Comment:
Suggestion: should there be an `assert` on the textArea `value` too?

Reply:
I think that makes sense! Went ahead and added